test/ReactViews/Custom/Chart/BottomDockChartSpec.tsx

```diff
@@ -47,23 +53,27 @@ describe("BottomDockChart", function () {
         updateIsSelectedInWorkbench: () => {},
         getColor: () => "#fff",
         pointOnMap: { latitude: -37.814, longitude: 144.96332 }
-      } as ChartItem
+      }
     ];
   });
 
-  it("renders all points on map for active chart items", function () {
+  it("renders all points on map for active chart items", async function () {
     act(() => {
-      testRenderer = TestRenderer.create(
+      createWithContexts(
+        viewState,
         <BottomDockChart
-          terria={terria}
+          height={100}
```

Comment:
Is `initialHeight` and `initialWidth` used, or should the existing parameters be renamed to `height` and `width` instead?

Reply:
It is used by `withParentSize` HOC